spkrepo/cli.py

```diff
@@ -1,87 +1,36 @@
-# -*- coding: utf-8 -*-
-import datetime
-import json
 import os
-import re
 import shutil
-import click
-
-from flask import current_app
-from flask.cli import FlaskGroup, AppGroup
 
-from spkrepo import create_app
-from spkrepo.ext import db
-from spkrepo.models import Architecture, Package, user_datastore
-from spkrepo.utils import populate_db
+import click
+from flask.cli import with_appcontext
 
-cli = FlaskGroup(create_app)
-@click.option("-c", "--config", help="config", required=False)
+from .ext import db
 
-def date_handler(obj):
-    return obj.isoformat() if hasattr(obj, "isoformat") else obj
 
+@click.group()
+def spkrepo():
+    """Spkrepo admin commands."""
 
-def pprint(obj):
-    print(json.dumps(obj, default=date_handler, sort_keys=True, indent=4))
 
-# User commands
-user_cli = AppGroup('user', help="Perform user actions")
-@user_cli.command('create')
+@spkrepo.command("create_user")
 @click.option("-u", "--username", help="username", default=None)
 @click.option("-e", "--email", help="email", default=None)
 @click.option("-p", "--password", help="password", default=None)
```

Comment:
missing the arguments --active and --confirmed

with those, additional db scripts are not required to login with this user.

Reply:
Oops, active = True and confirmed_at = datetime.datetime.now() by default